Misc/NEWS.d/next/Library/2024-05-21-19-10-30.gh-issue-115225.eRmfJH.rst

```diff
@@ -0,0 +1 @@
+Raise error on certain technically valid but pathological ISO 8601 strings passed to :meth:`datetime.time.fromisoformat` that were previously parsed incorrectly.
```

Comment:
If you want you can put (Patch by ...) with whatever name you are comfortable using here at the end of this file